extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+2. Including ths information prevents extraneous HTTP requests that would result in 404 errors.
+
+Availability takes the form of a bitstream with one bit per node in consideration. A 1 indicates that a tile/content/subtree is available at this node. Meanwhile, a 0 indicates that no tile/content/subtree is available at this node.
+
+For cases where all nodes are available or all nodes are unavailable, the `constant` property should be used instead of storing a full bitstream. For example, `constant: 0` indicates that all bits are 0 and no bitstream must be stored.
+
+Availability data is scoped to a subtree. This ensures that the size of each bitstream is bounded to a reasonable size.
+
+### Tile Availability
+
+**Tile availability** is a bitstream that determines which tiles exist within a subtree. There is one bit for each subtree node. A 1 indicates that a tile is available, while a 0 indicates that a tile is unavailable.
+
+![Tile Availability](figures/tile-availability.jpg)
+
+In the diagram above, colored nodes indicated available tile, while nodes with dashed outlines are unavailable tiles.
+
+If a tile is marked as available, more information about the tile, such as its content or children can be queried. If a tile is marked as unavailable, the tile must be skipped.
+
+### Content Availability
+
+**Content availability** is a bitstream that determines which tiles have an associated content file. Like tile availability, there is one bit for each subtree node. A 1 indicates a content file exists for this tile, while a 0 indicates that no content file exists.
+
+![Content Availability](figures/content-availability.jpg)
+
+The purpose of content availability is to check if a file exists before making a network request. If content is marked as unavailable, the network request for that file must be skipped.
+
+A content availability bit must only be set if the corresponding tile availability bit is set. Otherwise, it would be possible to specify content files that are not reachable by the tiles of the tileset. The content availability bitstream can be validated by checking that the following equation holds true:
+
+```
+contentAvailability & ~tileAvailability === 0
+```
+
+where `&` is the bitwise AND operation and `~` is the bitwise NOT operation.
+
+### Child Subtree Availability
+
+**Child subtree availability** is a bitstream that determines what subtrees can be reached from this subtree. There are `N` bits for every node in the bottom-most level of the subtree, where `N` is the branching factor of the tree. A 1 means there is a child subtree available at that position in the tree. Meanwhile, a 0 means there is no subtree available.
+
+![Child Subtree Availability](figures/subtree-availability.jpg)
+
+Child subtree availability is used to determine whether further subtree files exist before making network requests. If a child subtree availability bit is 0, any network request for that subtree must be skipped.
```

Comment:
I find this wording somewhat ambiguous. "further" isn't the right word here.

Reply:
An alternative phrasing could be:

```suggestion
Child subtree availability is used to determine whether files for child subtrees exist before making network requests. If a child subtree availability bit is 0, any network request for that subtree must be skipped.
```